src/KeyboardInputModal.js

```diff
@@ -49,15 +67,17 @@ class KeyboardInputModal extends React.Component<KeyboardInputModalProps, {}> {
             <Modal
                 onHide={this.props.onHide}
                 show={this.props.show}
+                aria-modal={true}
+                role="dialog"
                 dialogClassName='KeyboardInputModal'
             >
                 <Modal.Body className='KeyboardInputModal__content'>
                     <h2 className="KeyboardInputModal__content__title">
                         <FormattedMessage id='KeyboardInputModal.Title'/>
                     </h2>
-                    <div>
+                    <ul className="KeyboardInputModal__content__list">
```

Comment:
Seems like default ul style has padding left caused by "padding-inline-start: 40px". Because of this style, the list items seems not horizontally centred. Let's have a css rule for className KeyboardInputModal__content__list to remove the padding.

in KeyboardInputModal style sheet, 
```
.KeyboardInputModal__content__list {
padding-inline-start: 0;
}
```

Reply:
I did this a while back, but ended up removing it later in favour of an explicit `padding-left` on the content list once I got rid of the padding inherited from the bootstrap modal styles.